src/components/Contentful/RichTextItemsCentered.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<section-with-background-classic
+		:background-content="sectionBackground"
+		:vertical-padding="verticalPadding"
+	>
+		<template #content>
+			<kv-page-container>
+				<kv-grid class="tw-grid-cols-12 tw-text-center">
```

Comment:
We'll also need to add `:style="customGridStyles"` to this kv-grid. This will allow other uses of this content group where we can pass a setting of `align: left` and overwrite tw-text-center